bundle/src/bundler.rs

```diff
@@ -65,11 +74,24 @@ impl BundlerUtil {
             })?;
 
         if let Some(CodeOwners { ref path, .. }) = self.meta.base_props.codeowners {
-            let mut file = std::fs::File::open(path)?;
+            let mut file = File::open(path)?;
             tar.append_file("CODEOWNERS", &mut file)?;
             total_bytes_in += std::fs::metadata(path)?.len();
         }
 
+        if let Some(bep_parser) = self.bep_parser.as_ref() {
+            let mut bep_events_file = tempfile::tempfile()?;
+            bep_parser.bep_test_events().iter().for_each(|event| {
+                if let Err(e) = serde_json::to_writer(&bep_events_file, event) {
+                    log::error!("Failed to write BEP event: {}", e);
+                }
+            });
```

Comment:
nit, optional: `for_each` is for side-effects. IMO, `fold` or even a `for` loop might be more appropriate for showing intention

```suggestion
            let mut bep_events_file =
                bep_parser
                    .bep_test_events()
                    .iter()
                    .fold(tempfile::tempfile()?, |f, event| {
                        if let Err(e) = serde_json::to_writer(&f, event) {
                            log::error!("Failed to write BEP event: {}", e);
                        }
                        f
                    });
```